libraries/AP_Logger/LogFile.cpp

```diff
@@ -929,6 +929,22 @@ void AP_Logger::Write_VisualPosition(uint64_t remote_time_us, uint32_t time_ms,
     WriteBlock(&pkt_visualpos, sizeof(log_VisualPosition));
 }
 
+// Write visual velocity sensor data, velocity in NED meters per second
+void AP_Logger::Write_VisualVelocity(uint64_t remote_time_us, uint32_t time_ms, const Vector3f &vel, uint8_t reset_counter)
```

Comment:
Why aren't we passing through an accuracy estimate?

Reply:
thanks, added logging of the velocity error (or is it velocity noise?) in a follow up commit.